### PR TITLE
Update - add tips about echoserver image for apple m1 users

### PR DESCRIPTION
**Pull request by NicoleRobin**
the echoserver image in get started is not work for apple m1 user, i waste a little time here, so add a tip here to notify people who use apple m1.

detail discussion about this problem is here https://github.com/kubernetes/minikube/issues/12036#issuecomment-890100715

### CODE REVIEW

File: site/content/en/docs/start/_index.md [hunk 1]
```diff
@@ -534,7 +534,7 @@ minikube dashboard
 
 <h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">4</strong></span>Deploy applications</h2>
 
-Create a sample deployment and expose it on port 8080:
+Create a sample deployment and expose it on port 8080, if you are using apple m1, replace the image with `k8s.gcr.io/echoserver-arm:1.8`:
```
Comment (klaases) [inline site/content/en/docs/start/_index.md:537]: Suggestion: please capitalize proper nouns -`Apple M1` - for reference.

Comment (NicoleRobin) [reply site/content/en/docs/start/_index.md:537]: ok

Comment (NicoleRobin) [reply site/content/en/docs/start/_index.md]: > 

Done

Comment (NicoleRobin) [reply site/content/en/docs/start/_index.md]: It's a good suggestion. The modification has been completed

File: site/content/en/docs/start/_index.md [hunk 2]
```diff
@@ -534,7 +534,7 @@ minikube dashboard
 
 <h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">4</strong></span>Deploy applications</h2>
 
-Create a sample deployment and expose it on port 8080:
+Create a sample deployment and expose it on port 8080, if you are using Apple M1, replace the image with `k8s.gcr.io/echoserver-arm:1.8`:
```
Comment (afbjorklund) [inline site/content/en/docs/start/_index.md:537]: note that this is an armv7 (arm32) image, and not a arm64 image... It needs to be updated, to a newer nginx, for that.

Comment (afbjorklund) [reply site/content/en/docs/start/_index.md:537]:
```
REPOSITORY                  TAG       IMAGE ID       CREATED       SIZE
k8s.gcr.io/echoserver       1.4       a90209bb39e3   6 years ago   140MB
k8s.gcr.io/echoserver-arm   1.8       72565bf5bbed   5 years ago   85MB
```

Comment (afbjorklund) [reply site/content/en/docs/start/_index.md:537]: * https://github.com/kubernetes/minikube/issues/11107

**Pull request closed without merging**